### PR TITLE
Added Cordentity proposal

### PR DESCRIPTION
**Pull request by jeremiah**
Luxoft would like to bring Cordentity to Hyperledger. 

Signed-off-by: Jeremiah C. Foster <jfoster@luxoft.com>

### CODE REVIEW

**Comment by tkuhrt** [comment]
Hi, @jeremiah - It looks like your PR removes the README.md, LICENSE, and proposal-template.md. This should not be. It also has two proposals, one in the root directory and one in the labs directory. There should only be the one in the labs directory. Can you update this PR to only have one file changed, namely the new addition of labs/cordentity.md?

File: cordentity.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Lab Name
+Cordentity
+
+# Short Description
+The Cordentity project integrates Indy capabilities into the Corda platform.
```
Comment (tkuhrt) [inline cordentity.md]: Exchange any use of the word `project` with `lab` in the proposal.

**Comment by tkuhrt** [comment]
@dcmiddle : Can you confirm sponsorship?

**Comment by dcmiddle** [comment]
> @dcmiddle : Can you confirm sponsorship?

Yes, I confirm sponsoring this lab.

**Comment by jeremiah** [comment]
Didn't mean to close this. I'm having a fat-finger day.

File: labs/cordentity.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+open-source platform for self sovereign identity. This integration
+enables the development of decentralized applications on R3's best in
+class blockchain for business while utilizing the self-sovereign
+identity capabilities of Indy.
+
+# Scope of Lab
+
+Cordentity is a self-contained CorDapp that integrates Hyperledger
+Indy, for decentralized identity, with the R3 Corda Platform. This
+project creates interoperability of two purpose-built ledger
```
Comment (tkuhrt) [inline labs/cordentity.md]: change project to lab

**Comment by tkuhrt** [comment]
Thanks, @jeremiah, for the update. Thanks, @dcmiddle, for the confirmation. I made one other comment. Then we can have other @hyperledger-labs/lab-stewards weigh in on this lab.

**Comment by vipinsun** [comment]
LGTM

File: labs/cordentity.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+# Short Description
+The Cordentity app integrates Indy capabilities into the Corda platform.
+
+While Corda is suited for developing decentralized applications for
+managing complex inter-organizational workflows, Indy is the leading
+open-source platform for self sovereign identity. This integration
+enables the development of decentralized applications on R3's best in
+class blockchain for business while utilizing the self-sovereign
+identity capabilities of Indy.
```
[the remark this answers is not in the document]
Comment (jeremiah) [reply labs/cordentity.md]: It is marketing copy I'm afraid, marketing sent it to me thinking that the point of publishing at Hyperledger was mostly marketing. That said, I'm happy to tone it down a bit.

**Comment by tkuhrt** [comment]
@jeremiah : It looks like your last commits did not have the DCO sign off (`- s` on the `git commit`). Can you correct this using the steps under the [DCO details](https://github.com/hyperledger-labs/hyperledger-labs.github.io/pull/57/checks?check_run_id=35275505)?

**Comment by jeremiah** [comment]
I edited via the web interface but of course that doesn't sign things for me. I had to use my personal address but everything is signed.

**Comment by tkuhrt** [comment]
@jeremiah : I notice that you provide a certain branch for existing repositories. I am unaware of how we could transfer only a branch to another organization without a manual process. Since the other branches appear to have DCO missing, I will create a new lab, which you will then need to push the source code to.

**Comment by tkuhrt** [comment]
Created https://github.com/hyperledger-labs/cordentity and invited all initial committers to join the team.

**Comment by jeremiah** [comment]
On Tue, Nov 27, 2018 at 12:46:56PM -0800, Tracy Kuhrt wrote:
>    @jeremiah : I notice that you provide a certain branch for existing
>    repositories. I am unaware of how we could transfer only a branch to
>    another organization without a manual process. Since the other branches
>    appear to have DCO missing, I will create a new lab, which you will then
>    need to push the source code to.

Okay.

**Pull request merged**